src/transform/DimImagesTransform.js

```diff
@@ -1,27 +1,40 @@
 import './DimImagesTransform.css'
 const CLASS = 'pagelib_dim_images'
 
-// todo: only require a Document
 /**
- * @param {!Window} window
+ * @param {!Document} document
  * @param {!boolean} enable
  * @return {void}
  */
-const dim = (window, enable) => {
-  window.document.querySelector('html')
-    .classList[enable ? 'add' : 'remove'](CLASS)
+const dimImages = (document, enable) => {
+  document.querySelector('html').classList[enable ? 'add' : 'remove'](CLASS)
 }
 
-// todo: only require a Document
 /**
+ * @deprecated Use dimImages instead, which only requires a Document
+ * @param {!Window} window
+ * @param {!boolean} enable
+ * @return {void}
+ */
+const dim = (window, enable) => dimImages(window.document, enable)
+
+/**
+ * @param {!Document} document
+ * @return {boolean}
+ */
+const isDimImagesOn = document => document.querySelector('html').classList.contains(CLASS)
+
+/**
+ * @deprecated Use isDimImagesOn instead, which only requires a Document
```

Comment:
I'd say this would preferably be removed, too, if possible.

Reply:
Same as the other deprecated function above.